pkgs/development/python-modules/wxPython/4.0.nix

```diff
@@ -22,14 +35,43 @@ buildPythonPackage rec {
     sha256 = "35cc8ae9dd5246e2c9861bb796026bbcb9fb083e4d49650f776622171ecdab37";
   };
 
-  nativeBuildInputs = [
-    pkgconfig
-  ];
+  doCheck = false;
 
-  buildInputs = [
-    gtk3 libjpeg libtiff SDL gst-plugins-base libnotify freeglut xorg.libSM
-    which
-  ];
+  nativeBuildInputs = [ pkgconfig which doxygen wxGTK ];
+
+  buildInputs = [ libjpeg libtiff SDL
+      gst-plugins-base libnotify freeglut xorg.libSM ncurses
+      requests libpng gstreamer libX11
+      pathlib2
+      (wxGTK.gtk)
+  ]
+    ++ lib.optional openglSupport pyopengl;
+
+  hardeningDisable = [ "format" ];
```

Comment:
Why?

Reply:
I guess this one should be self-explanatory as long as reader understands what "format" hardening option stands for. There probably is an unsafe printf/scanf expression in this code.

Reply:
Think I copied from https://github.com/NixOS/nixpkgs/blob/3c6ec09222efe40461a5536e5df26f3e7031ae04/pkgs/development/python-modules/wxPython/3.0.nix and it seemed to work